src/module/addRemove.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-loop-func */
+let todoList = [];
+let isEditing = false;
+let todoEdit = null;
+
+const saveData = () => {
+  localStorage.setItem('todoslist', JSON.stringify(todoList));
+};
+
+const getStorageData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('todoslist'));
+  if (localFormData == null) {
+    todoList = [];
+  } else {
+    todoList = localFormData;
+  }
+};
+
+const editTodoList = (todo) => {
+  isEditing = true;
+  todoEdit = todo;
+  const desc = document.querySelector('.taskList');
+  desc.value = todo.description;
+  desc.focus();
+};
+
+const removeTodoList = (indexID) => {
+  todoList = todoList.filter((ind) => ind.index !== indexID);
+  todoList = todoList.map(
+    (todo, index) => (
+      { completed: todo.completed, description: todo.description, index: index + 1 }
+    ),
+  );
+  displayToDo();
+};
+
+const toggleToDoStatus = (todo) => {
+  todoList = todoList.map((todoItem) => {
+    if (todoItem.index === todo.index) {
+      return { ...todo, completed: !todo.completed };
```

Comment:
This code is not clear, please create a variable and assign to it the value to return in this method.